packages/core-flows/src/store/workflows/create-stores.ts

```diff
@@ -0,0 +1,13 @@
+import { StoreDTO, CreateStoreDTO } from "@medusajs/types"
+import { WorkflowData, createWorkflow } from "@medusajs/workflows-sdk"
+import { createStoresStep } from "../steps"
+
+type WorkflowInput = { storesData: CreateStoreDTO[] }
```

Comment:
todo: snake_case or rename
```suggestion
type WorkflowInput = { data: CreateStoreDTO[] }
```